pennylane/pauli/pauli_arithmetic.py

```diff
@@ -364,7 +376,11 @@ def __missing__(self, key):
 
     def __add__(self, other):
         """Add two Pauli sentence together by iterating over the smaller
-        one and adding its terms to the larger one."""
+        one and adding its terms to the larger one.
+
+        Empty Pauli sentences are treated as 0. The non-empty Pauli sentence
```

Comment:
Personally I prefer the phrasing above that spells it out "are treated as the additive identity (i.e. 0 * Identity on all wires)" instead of just "are treated as 0".

Reply:
Added 👍🏼